website/docs/add-secure-apps/providers/ssf/index.md

```diff
@@ -1,30 +1,39 @@
 ---
 title: Shared Signals Framework (SSF) Provider
 sidebar_label: SSF Provider
+tags: [ssf, backchannel, provider, "security event tokens"]
 authentik_version: "2025.2.0"
 authentik_enterprise: true
 authentik_preview: true
 ---
 
-Shared Signals Framework (SSF) is a common standard for sharing asynchronous real-time security signals and events across multiple applications and an identity provider. The framework is a collection of standards and communication processes, documented in a [specification](https://openid.net/specs/openid-sharedsignals-framework-1_0-ID3.html). SSF leverages the APIs of the application and the IdP, using privacy-protected, secure webhooks.
+Shared Signals Framework (<abbr>SSF</abbr>) is a collection of standards for sharing asynchronous security signals and events across multiple applications and an identity provider. SSF leverages the APIs of the application and the IdP, using privacy-protected, secure webhooks.
 
-## About Shared Signals Framework
+## What are security signals?
 
-In authentik, an SSF provider allows applications to subscribe to certain types of security signals (which are then translated into SETs, or Security Event Tokens) that are captured by authentik (the IdP), and then the application can respond to each event. In this scenario, authentik acts as the _transmitter_ and the application acts as the _receiver_ of the events.
+An SSF provider allows applications to subscribe to _security signals_, translating them into Security Event Tokens (<abbr>SET</abbr>) that are captured by the IdP. In this arrangement, authentik acts as an mediator from an external event _transmitter_, ultimately with a specific application acting as the _receiver_ of the events.
 
-Events in authentik that are tracked via SSF include when an MFA device is added or removed, logouts, sessions being revoked by Admin or user clicking logout, or credentials changed.
+### Transmitter and receiver
```

Comment:
Thanks for the new section, well-written! My only concern is that, if I understand correctly, there are cases where this can be reversed, and the App is the transmitter. That's why I have phrases like on line 13... "in authentik"... to be clear that we are describing how authentik works, not generalizing.